### ISSUE TITLE
feature: do not require a changes file

### ISSUE THREAD

**Issue by jannes** [houseabsolute/actions-rust-release#3]
I would love to use this action but without a changes file.
Right now it looks like it is required, even though the readme suggests it is not when `extra-files` is used.

**Comment by jannes** [comment]
I think the current behavior should be considered a bug given the description in README.

I only specified `extra-files: README.md` and my action failed with 
```
Changes file 'Changes.md' does not exist. at /home/runner/work/_actions/houseabsolute/actions-rust-release/v0.0.2/make-archive.pl line 38.
```
However in the description it says if `extra-files` is specified, a changes file needs to be passed explicitly.

**Comment by jannes** [comment]
I solved this by using `changes-file: null` for my own use case. Maybe just an update to the description in README.me is enough.

**Comment by autarch** [comment]
Yeah, this is definitely a bug. It shouldn't error out if it's using the default value and that file doesn't exist. I'm not sure if there's a way to detect that the default is being used, as opposed to someone passing `Changes.md` explicitly.  I'll think about the best way to address this.

**Comment by autarch** [comment]
This is fixed in v0.0.3. If you set `extra-files`, it ignored anything set in `changes-file`. I updated the docs to clarify this.
